wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -903,4 +921,124 @@ public static function setIndianStateOptions(string $entity, string $field, arra
 
   }
 
+  /**
+   * This hook is called after the database write on a custom table.
+   *
+   * @param string $op
+   *   The type of operation being performed.
+   * @param string $objectName
+   *   The custom group ID.
+   * @param int $objectId
+   *   The entityID of the row in the custom table.
+   * @param object $objectRef
+   *   The parameters that were sent into the calling function.
+   */
+  public static function mailNotificationToMmt($op, $groupID, $entityID, &$params) {
+    if ($op !== 'create') {
+      return;
+    }
+
+    if (!($goonjField = self::findOfficeId($params))) {
+      return;
+    }
+
+    $goonjFieldId = $goonjField['value'];
+    $vehicleDispatchId = $goonjField['entity_id'];
+
+    $collectionSourceVehicleDispatch = EckEntity::get('Collection_Source_Vehicle_Dispatch', FALSE)
+      ->addSelect('Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id')
+      ->addWhere('id', '=', $vehicleDispatchId)
+      ->execute()->first();
+
+    $collectionCampId = $collectionSourceVehicleDispatch['Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id'];
+
+    $coordinators = Relationship::get(FALSE)
+      ->addWhere('contact_id_b', '=', $goonjFieldId)
+      ->addWhere('relationship_type_id:name', '=', self::MATERIAL_RELATIONSHIP_TYPE_NAME)
+      ->addWhere('is_current', '=', TRUE)
+      ->execute()->first();
+
+    $mmtId = $coordinators['contact_id_a'];
+
+    if (empty($mmtId)) {
+      return;
+    }
+
+    $email = Email::get(FALSE)
+      ->addSelect('email', 'contact_id.display_name')
+      ->addWhere('contact_id', '=', $mmtId)
+      ->execute()->single();
+
+    $mmtEmail = $email['email'];
+    $contactName = $email['contact_id.display_name'];
+
+    // Email to material management team member.
+    $mailParams = [
+      'subject' => 'New Entry For Matrial Dispatch Notification',
+      'from' => 'urban.ops@goonj.org',
+      'toEmail' => $mmtEmail,
+      'replyTo' => 'urban.ops@goonj.org',
+      'html' => self::goonjcustom_material_management_email_html($mmtId, $contactName, $collectionCampId),
+        // 'messageTemplateID' => 76, // Uncomment if using a message template
+    ];
+    $result = \CRM_Utils_Mail::send($mailParams);
+
+    $updateMmtId = EckEntity::update('Collection_Source_Vehicle_Dispatch', FALSE)
+      ->addValue('Acknowledgement_For_Logistics.Filled_by', $mmtId)
+      ->addWhere('Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id', '=', $collectionCampId)
+      ->execute();
+
+  }
+
+  /**
+   *
+   */
+  public static function goonjcustom_material_management_email_html($mmtId, $contactName, $collectionCampId) {
+    $homeUrl = get_home_url();
```

Comment:
[`get_home_url`](https://developer.wordpress.org/reference/functions/get_home_url/) is a WordPress function. In Civi extension we should avoid using the functions of the underlying CMS because that will limit the extension compatibility with other CMS.

A similar example is here where I have extracted the path of uploads directory from Civi Config object

https://github.com/ColoredCow/goonj/blob/eb968c52f445da9083001ea57ab16c114c53ecfe/wp-content/civi-extensions/goonjcustom/Civi/CollectionBaseService.php#L344-L345